src/react/utils.js

```diff
@@ -653,8 +659,7 @@ export function getProperty(
 ): Value {
   if (object instanceof AbstractObjectValue) {
     if (object.values.isTop()) {
-      // fallback to the original Get implementation
-      return Get(realm, object, property);
+      return realm.intrinsics.undefined;
```

Comment:
I don’t understand why this is safe. It seems to say “if we read from an object that we don’t know about, let’s just say the property is undefined”. Why is that correct?

Reply:
This is an internal getter, we shouldn't be falling back to the original Get in any case, that was my mistake. If we fallback, we might as well just have done the original Get to begin with.